analysis.tex

```diff
@@ -1,6 +1,34 @@
 \section{Analysis}
 
-\begin{theorem}
+\begin{theorem}[Correctness]
+  The Proof-of-Burn protocol $\Pi$ of Section~\ref{section:construction} is \emph{correct}.
+\end{theorem}
+\begin{proof}
+  We need to prove that $\forall \kappa, t. \BurnVerify(1^\kappa, t, \GenBurnAddress(1^\kappa, t)) = \textsf{true}$.
+
+  Based on Algorithm~\ref{alg.construction}, $\BurnVerify(1^\kappa, t, \GenBurnAddress(1^\kappa, t)) = \textsf{true}$ if and only if $\GenBurnAddress(1^\kappa, t) = \GenBurnAddress(1^\kappa, t)$, which is a tautology as $\GenBurnAddress$ is deterministic.
```

Comment:
Would avoid the word "tautology" for things that are not logical tautologies (such as "p or not p"). We can just say "it holds, as $\GenBurnAddress$ is deterministic."